README.md

```diff
@@ -115,6 +115,10 @@ Submissions will be analyzed for correctness, performance, and stability.
 
 All submissions are required to be open-sourced. The judging team will review the source code, and will build the executable to be judged from this source code according to the instructions provided in the submission documentation. Note that if your solution requires particular build flags or a particular compiler version, it should be referenced in the build documentation.
 
+### Functionality
+
+To be suitable for the judging process, each app should copy as closely as possible the behavior of the provided test harness. In particular, its input and output behavior should be identical. The app should be able to take in two files as input: one file containing a list of n scalar vectors, one per line, and another file containing a list of n curve point vectors, one per line, where the i-th scalar vector is the same length as the i-th curve point vector. The app should sequentially perform n separate MSM instances, one per vector, and output two files. One file should contain the n output curve elements, one line per MSM instance. The other should contain the n wall-clock times taken to execute the MSM instances, one per line, in addition to one more line containing the mean time across all n previous times. 
```

Comment:
We should probably also mention the input and output encodings. Also the time unit for the outputs.

```suggestion
To be suitable for the judging process, each app should copy as closely as possible the behavior of the provided test harness. In particular, its input and output behavior should be identical. The app should be able to take in two files as input: one file containing a list of n scalar vectors, one per line, and another file containing a list of n curve point vectors, one per line, where the i-th scalar vector is the same length as the i-th curve point vector. The app should sequentially perform n separate MSM instances, one per vector, and output two files. One file should contain the n output curve elements, one line per MSM instance. The other should contain the n wall-clock times, in milliseconds, taken to execute the MSM instances, one per line, in addition to one more line containing the mean time across all n previous times. All point and point vectors should encoding equivalently to the arkworks uncompressed point representation at version `0.3.0`. If there are any questions, please refer to the reference implementation or contact the judging team listed above.
```